taxi/views.py

```diff
@@ -87,3 +89,29 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
```

Comment:
Using .all before prefetch_related is redundant because prefetch_related already retrieves all related objects.